tests/test_box_Box.py

```diff
@@ -208,21 +208,21 @@ def test_vectors(self):
 
     def test_periodic(self):
         box = freud.box.Box(1, 2, 3, 0, 0, 0)
-        false = [False, False, False]
-        true = [True, True, True]
-        self.assertEqual(box.periodic, true)
+        falses = [False, False, False]
+        trues = [True, True, True]
```

Comment:
I would prefer these written either using `np.[zeros|ones](3, dtype=bool)` or `[False|True]*3`, but again I'll leave it to your discretion if you actually want to make the change.

Reply:
Actually, I take it back. Can you remove these arrays entirely and just change all the tests to `npt.assert_array_equal(box.periodic, True)`? That looks much better.